### ISSUE TITLE
result

### ISSUE THREAD

**Issue by LoveSimons** [pkuCactus/BDCN#17]
Hello, I am very interested in your work. I ran the test.py script and it passed. The result is as follows:
[1]runfile('/home/renxue/BDCN/test.py', wdir='/home/renxue/BDCN')
Time.struct_time(tm_year=2019, tm_mon=7, tm_mday=7, tm_hour=9, tm_min=15, tm_sec=6, tm_wday=6, tm_yday=188, tm_isdst=0)
Bdcn.py:199: UserWarning: nn.init.constant is now deprecated in favor of nn.init.constant_.
   Nn.init.constant(param, 0.080)
(200, 200)
/home/renxue/anaconda3/envs/psenet/lib/python2.7/site-packages/torch/nn/functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
   Warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
383.229988098
Overall Time use: 383.677251816
There is no visual result in the result/fuse folder. How can I solve this problem?
I look forward to your reply and wish you a happy life.

**Comment by jannctu** [comment]
Hi, I run and retrain this code, and it's work fine. 
His pretrained model ODS : 0.817 
My retrain model ODS : 0.805

**Comment by LoveSimons** [comment]
> Hi, I run and retrain this code, and it's work fine.
> His pretrained model ODS : 0.817
> My retrain model ODS : 0.805

Thank you very much for your reply. The three evaluation indicators mentioned in the article: ODS, OIS and AP, are there tools that can directly get these three results, because the code does not involve evaluation indicators. Can you share the tools for calculating the evaluation indicators?
look forward to your reply

**Comment by jannctu** [comment]
Yes, the evaluation code is written in Matlab. That's part of the Structured Forest paper. But, that code is also used in HED and RCF papers. it's good to read all those (HED, RCF) papers first to get better understanding.

**Comment by LoveSimons** [comment]
> Yes, the evaluation code is written in Matlab. That's part of the Structured Forest paper. But, that code is also used in HED and RCF papers. it's good to read all those (HED, RCF) papers first to get better understanding.

Thank you very much for your reply. Based on your suggestions, I will study further and wish you a happy life.

**Comment by qianyu220** [comment]
> Hello, I am very interested in your work. I ran the test.py script and it passed. The result is as follows:
> [1]runfile('/home/renxue/BDCN/test.py', wdir='/home/renxue/BDCN')
> Time.struct_time(tm_year=2019, tm_mon=7, tm_mday=7, tm_hour=9, tm_min=15, tm_sec=6, tm_wday=6, tm_yday=188, tm_isdst=0)
> Bdcn.py:199: UserWarning: nn.init.constant is now deprecated in favor of nn.init.constant_.
>    Nn.init.constant(param, 0.080)
> (200, 200)
> /home/renxue/anaconda3/envs/psenet/lib/python2.7/site-packages/torch/nn/functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
>    Warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
> 383.229988098
> Overall Time use: 383.677251816
> There is no visual result in the result/fuse folder. How can I solve this problem?
> I look forward to your reply and wish you a happy life.

Hi, I wonder did you solve this problem? I have the same problem that I can't find any result in the folder. Thanks!
